app/src/main/java/org/wikipedia/talk/TalkReplyActivity.kt

```diff
@@ -462,14 +479,25 @@ class TalkReplyActivity : BaseActivity(), LinkPreviewDialog.Callback, UserMentio
 
     override fun onBackPressed() {
         setResult(RESULT_BACK_FROM_TOPIC)
+        if (viewModel.isFromDiff) {
+            sendPatrollerExperienceEvent("publish_back", "pt_warning_messages")
+        }
         if (viewModel.isNewTopic && (!binding.replySubjectText.text.isNullOrEmpty() ||
                     binding.replyInputView.editText.text.isNotEmpty())) {
+            if (viewModel.isFromDiff) {
+                sendPatrollerExperienceEvent("publish_back", "pt_warning_messages")
+            }
```

Comment:
Is this necessary since we already have the same event above?

Reply:
Removed this